baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/StrictUnusedVariable.java

```diff
@@ -124,7 +128,7 @@
  */
 @AutoService(BugChecker.class)
 @BugPattern(
-        altNames = {"unused", "UnusedVariable"},
+        altNames = "unused",
```

Comment:
Should we also remove "unused"? [It is an alt name for the UnusedVariable check](https://github.com/google/error-prone/blob/master/core/src/main/java/com/google/errorprone/bugpatterns/UnusedVariable.java#L120), and users may use `-Xep:unused:ERROR` thinking they are disabling the `UnusedVariable` check, but they also disable this?

Reply:
That should be unlikely as it's not the name of an upstream error-prone check, or the intended use of the enablement options. I'd prefer to minimize out workaround as much as possible since it may not be 100% equivalent to the previous alternate name behavior.

Reply:
Ok, sounds good.